### ISSUE TITLE
Don't reissue releases all the time

### ISSUE THREAD

**Issue by jengelh** [gap-packages/HeLP#21]
```
59a9d6fd4dbba43c765d51c012f498cf  HeLP-4.0.tar.gz (March 4)
2b11d4b34976ae36e8c498fd91f839a6  HeLP-4.0.tar.gz (March 15)
```
This is really bad, because now, the help-4.0 shipped in some Linux distribution does not match with the "4.0" in upstream. And this is already like the fifth time 4.0 was reissued. If you *have to* make new releases, please given *each* of them *new* numbers.

**Comment by fingolfin** [comment]
Sorry for the inconvenience. However, releases for this project (and all other GAP packages under `gap-packages` are only "official" when they appear on the website, e.g. https://gap-packages.github.io/HeLP/ 

I assume you are somehow monitoring https://github.com/gap-packages/HeLP/releases for new releases. Please don't do that, the results will not match with what is in GAP or in other package distros.

**Comment by jengelh** [comment]
>"official" when they appear on the website, e.g. https://gap-packages.github.io/HeLP/

Websites are not really machine-readable.

GH has a checkbox you could use for non-"official":

![1](https://github.com/user-attachments/assets/083091e9-c82c-4c64-bb50-36f6d6d0677f)

**Comment by fingolfin** [comment]
Yeah, but then someone has to invest time to update the tooling to handle yet another thing (those releases are created by a script). Unfortunately saying we are "understaffed" is an understatement. 

If you want a machine readable version of which package versions are officially considered as released, you could parse https://github.com/gap-system/PackageDistro/releases/download/latest/package-infos.json.gz
